packages/components/src/components/dropdown/readme.md

```diff
@@ -156,9 +156,10 @@ mi-dropdown::part(button) {
 
 ## Events
 
-| Event    | Description                                      | Type               |
-| -------- | ------------------------------------------------ | ------------------ |
-| `change` | Triggers an event when the selection is changed. | `CustomEvent<any>` |
+| Event     | Description                                      | Type                |
+| --------- | ------------------------------------------------ | ------------------- |
+| `change`  | Triggers an event when the selection is changed. | `CustomEvent<any>`  |
+| `cleared` |                                                  | `CustomEvent<void>` |
```

Comment:
The JSDoc description was not transferred here in the build process. Please re-trigger a build to make it happen.